src/mono/sample/wasi/native/Wasi.Native.Sample.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <RuntimeIdentifier>wasi-wasm</RuntimeIdentifier>
+    
+    <TargetOs>wasi</TargetOs>
+    <WasmBuildNative>true</WasmBuildNative>
+    <WasmNativeStrip>false</WasmNativeStrip>
+    <IsBrowserWasmProject>false</IsBrowserWasmProject>
+    <WasmSingleFileBundle>true</WasmSingleFileBundle>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <NativeFileReference Include="local.c" />
```

Comment:
Do I understand correctly that all symbols in this sample are statically linked inside dotnet.wasm ? If so, should we also have sample that makes them external ?

Reply:
This probably need to happen in the tests rather than the sample, I'm pretty sure the smoke tests run the samples and the import wouldn't resolve.